### ISSUE TITLE
Showcase: Add a history of previous themes activated or tried

### ISSUE THREAD

**Issue by catkit** [Automattic/wp-calypso#23204]
While helping users on Live Chat a question that we get very often is "
what was my previous theme before this one please? I don't remember the name"
So I thought it would be useful to display somewhere in the showcase a history of previous themes and themes that were tried by the user.

**Comment by kathrynwp** [comment]
I agree this would be a great enhancement - it comes up quite often in the forums.

**Comment by catkit** [comment]
I still find this request relevant

**Comment by kathrynwp** [comment]
And I still completely agree, it  comes up all the time in support.

**Comment by wpericam** [comment]
Couldn't remember which theme to go back to 2435652-zd

**Comment by Aurorum** [comment]
PR in #40791 to add a button to the previous theme, which I think will resolve the sentiment of this issue in a cleaner and easier way. :) 

<img width="1246" alt="Screenshot 2020-04-05 at 10 35 09" src="https://user-images.githubusercontent.com/43215253/78471454-4ef0f480-7729-11ea-8b08-8352035a2fe8.png">

**Comment by kwight** [comment]
We need a more holistic approach to this problem, as opposed to just the button – we'll work on this though it seem like high ROI.

**Comment by kwight** [comment]
A first step could be the Undo Theme option.
